picopass/scenes/picopass_scene_read_card_success.c

```diff
@@ -190,6 +195,10 @@ bool picopass_scene_read_card_success_on_event(void* context, SceneManagerEvent
             consumed = scene_manager_search_and_switch_to_another_scene(
                 picopass->scene_manager, PicopassSceneStart);
         }
+    } else if(event.type == SceneManagerEventTypeBack) {
```

Comment:
Based on Seader, since we have a left "Retry", the back button should just be a way of going back to the start menu

Reply:
LGTM

Steps to reproduce bug before fix:
- Emulate provided iCLASS SE dump file using a pm3.
- `picopass read > save partial > emulate partial file > collect reader mac > re-read SE emulation > config card detected`
- iCLASS SE dump file provided below for testing



[iclassSE.json](https://github.com/flipperdevices/flipperzero-good-faps/files/13710757/iclassSE.json)